docker/Dockerfile.dev.ui

```diff
@@ -19,7 +19,7 @@ COPY ./keep-ui/ /app
 RUN npm install
 # Install next globally and create a symlink
 RUN npm install -g next
-RUN ln -s /usr/local/lib/node_modules/next/dist/bin/next /usr/local/bin/next
+RUN ln -s /usr/local/lib/node_modules/next/dist/bin/next /usr/local/bin/next || echo "next binary already linked to bin"
```

Comment:
I would put it to separate commit, as it don't seems to be related to this feature.